ivy_tests/test_ivy/test_frontends/test_numpy/test_fft/test_discrete_fourier_transform.py

```diff
@@ -189,3 +189,24 @@ def test_numpy_rfftfreq(n, sample_rate, frontend, test_flags, fn_tree, on_device
         n=n,
         d=d,
     )
+
+@handle_frontend_test(
```

Comment:
lint is failing because there should be 2  lines space between tests function but your test function and other function has 1 line difference so make it

Reply:
Lint test still fails, do I need pre-commit installed, for some reason pre-commit does not allow me to push changes I get an error:
![Pre-Commit_Error](https://github.com/unifyai/ivy/assets/94248626/3e57507b-2c14-445f-a4ca-72149f463748)
So I had to uninstall pre-commit with its hooks,